### PR TITLE
Chapter 8: Basic Login

### PR DESCRIPTION
**Pull request by hieuvc-2823**
-basic login
branch_8

### CODE REVIEW

**Comment by hieuvc-2823** [comment]
Screenshot from 2021-11-29 13-41-04

**Comment by hieuvc-2823** [comment]
Screenshot from 2021-12-06 08-32-28

**Comment by hieuvc-2823** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <header class="navbar navbar-fixed-top navbar-inverse">
   <div class="container">
-    <%= link_to t("app_name"), static_pages_home_path , id: "logo" %>
+    <%= link_to t("app_name") , "/" , id: "logo" %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb]: Sao đổi code thế bạn

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -9,9 +9,32 @@
           <li>
             <%= link_to t("help.index"), static_pages_help_path %>
           </li>
-          <li>
-            <%= link_to t("log_in"), "#" %>
-          </li>
+          <% if logged_in? %>
+            <li>
+              <%= link_to t("users") , users_path %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb]:
```suggestion
              <%= link_to t("users"), users_path %>
```

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -9,9 +9,32 @@
...
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                <%= t("account") %> <b class="caret"></b>
+              </a>
+              <ul class="dropdown-menu">
+                <li>
+                  <%= link_to t("profile"), current_user %>
+                </li>
+                <li>
+                  <%= link_to t("setting") , edit_user_path(current_user) %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb]:
```suggestion
                  <%= link_to t("setting"), edit_user_path(current_user) %>
```

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -9,9 +9,32 @@
...
+                </li>
+                <li class="divider"></li>
+                <li>
+                  <%= link_to t("log_out") , logout_path %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb]:
```suggestion
                  <%= link_to t("log_out"), logout_path %>
```

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -9,9 +9,32 @@
...
+                </li>
+              </ul>
+            </li>
+          <% else %>
+            <li>
+              <%= link_to t("log_in") , login_path %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb]:
```suggestion
              <%= link_to t("log_in"), login_path %>
```

File: config/routes.rb [hunk 6]
```diff
@@ -3,5 +3,8 @@
   get "static_pages/help"
   get "/signup", to: "users#new"
   post "/signup", to: "users#create"
+  get  "/login", to: "sessions#new"
```
Comment (vanvtt-0952) [inline config/routes.rb]:
```suggestion
  get "/login", to: "sessions#new"
```

**Comment by hieuvc-2823** [comment]
Screenshot from 2021-12-06 08-54-30

**Comment by hieuvc-2823** [comment]
ready

**Comment by vanvtt-0952** [comment]
commented
vẫn 2 commit thế kia, chém chú giờ

**Comment by hieuvc-2823** [comment]
Screenshot from 2021-12-06 10-02-57

**Comment by hieuvc-2823** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**